src/js/components/DataTable/doc.js

```diff
@@ -305,7 +321,12 @@ export const themeDoc = {
     defaultValue: "{ horizontal: 'small', vertical: 'xsmall' }",
   },
   'dataTable.header': {
-    description: 'Styles for the header.',
+    description: 'Styles for the header cells.',
+    type: 'object',
+    defaultValue: '{}',
+  },
+  'dataTable.headerGroup': {
+    description: 'Styles for the header group cells.',
```

Comment:
Is it Box styling? Table styling? we should be explicit about which styles the theme support.